.jenkins/build_docker_images.Jenkinsfile

```diff
@@ -1,12 +1,12 @@
 @Library("OpenEnclaveCommon") _
-oe = new jenkins.common.Openenclave()
+dcap = new jenkins.shared.Dcap()
 
 // The below timeout is set in minutes
 GLOBAL_TIMEOUT = 240
 
-OETOOLS_REPO = "https://oejenkinscidockerregistry.azurecr.io"
-OETOOLS_REPO_CREDENTIAL_ID = "oejenkinscidockerregistry"
-OETOOLS_DOCKERHUB_REPO_CREDENTIAL_ID = "oeciteamdockerhub"
+OETOOLS_REPO = "https://dcapdockerciregistry.azurecr.io"
+OETOOLS_REPO_CREDENTIAL_ID = "dcapdockerciregistry"
+//OETOOLS_DOCKERHUB_REPO_CREDENTIAL_ID = "oeciteamdockerhub"
```

Comment:
Please remove instead of commenting out - can bring it back in the commit history if we need

Reply:
Yes, I will remove them.